cli/src/cli_util.rs

```diff
@@ -3099,7 +3099,7 @@ pub struct GlobalArgs {
 
 #[derive(clap::Args, Clone, Debug)]
 pub struct EarlyArgs {
-    /// When to colorize output (always, never, debug, auto)
+    /// When to colorize output
```

Comment:
I still think displaying the options has some value.

Reply:
Without deleting this, `jj -h` would result in
```
      --color <WHEN>                 When to colorize output (always, never, debug, auto) [possible values: always, never, debug, auto]
```

Reply:
ok, then it is fine to remove.